nextflow_schema.json

```diff
@@ -88,7 +88,7 @@
                     "type": "boolean",
                     "description": "Skip FastQC"
                 },
-                "skip_emptydrops": {
+                "skip_cellbender": {
```

Comment:
I would keep the old parameter but set `"deprecated":true` and write an errorMessage entry, see https://nextflow-io.github.io/nf-schema/latest/nextflow_schema/nextflow_schema_specification/#deprecated

Reply:
addressing in addressing in: https://github.com/nf-core/scrnaseq/pull/443